dev/tools/bugzilla2github

```diff
@@ -0,0 +1,914 @@
+
+    if url[0] == "/":
+        u = "%s%s" % (github_url, url)
+    else:
+        u = "%s/repos/%s/%s/%s" % (github_url, github_owner, github_repo, url)
+
+    d = {}
+    # Copy fields into the data
+    for field in fields:
+        if field not in avs:
+            print "Error posting filed %s to %s" % (field, url)
+            exit(1)
+        d[field] = avs[field]
+
+    # TODO: debug
+    # print "POST: " + u
+    # print "DATA: " + json.dumps(d)
+
+    if force_update:
+        return requests.post(u, params = { "access_token": github_token },
+                                data = json.dumps(d))
+    else:
+        if not github_post.warn:
+            print "Skipping POST... (use -f to force updates)"
+            github_post.warn = True
+        return True
+
+github_post.warn = False
+
+
+def github_label_create(label):
+    if not github_get("labels/" + label):
+        print "\tcreating label '%s' on GitHub..." % label
+        r = github_post("labels", {
+            "name": label,
+            "color": "0"*6,
+        }, ["name", "color"])
+        if not r:
+            print "Error creating label %s: %s" % (label, r.headers)
+            exit(1)
+
+
+def github_labels_check(issues):
+    global force_update
+
+    labels_set = set()
+    for id in issues:
+        for label in issues[id]["labels"]:
+            labels_set.add(label)
+
+    for label in labels_set:
+        if github_get("labels/" + label):
+            print "\tlabel '%s' exists on GitHub" % label
+        else:
+            if force_update:
+                github_label_create(label)
+            else:
+                print "WARNING: label '%s' does not exist on GitHub" % label
+
+
+def github_assignees_check(issues):
+    a_set = set()
+    for id in issues:
+        if "assignee" in issues[id]:
+            a_set.add(issues[id]["assignee"])
+
+    for assignee in a_set:
+        if not github_get("/users/" + assignee):
+            print "Error checking user '%s' on GitHub" % assignee
+            exit(1)
+        else:
+            print "Assignee '%s' exists" % assignee
+
+
+def github_issue_exist(number):
+    if github_get("issues/%d" % number):
+        return True
+    else:
+        return False
+
+
+def github_issue_get(number):
+    req = github_get("issues/%d" % number)
+    if not req:
+        print "Error getting GitHub issue #%d: %s" % (number, req.headers)
+        exit(1)
+
+    return req.json()
+
+
+def github_issue_append(issue):
+    global github_owner, github_repo, github_token
+    params = { "access_token": github_token }
+    headers = { "Accept": "application/vnd.github.golden-comet-preview+json" }
+    print "\tappending a new issue on GitHub..."
+    u = "https://api.github.com/repos/%s/%s/import/issues" % (github_owner, github_repo)
+    comments = issue.pop("comments")
+    # We can't assign people which are not in the organization / collaborators on the repo
+    if github_owner != "coq":
```

Comment:
Also, I think you should possibly list the assigned people in the text of the bug report, if you can't actually assign the bug to them.